Formula/f/fcft.rb

```diff
@@ -0,0 +1,51 @@
+class Fcft < Formula
+  desc "Simple library for font loading and glyph rasterization"
+  homepage "https://codeberg.org/dnkl/fcft"
+  url "https://codeberg.org/dnkl/fcft/archive/3.1.8.tar.gz"
+  sha256 "f48c793f354b8be95477e475dde7c98ac9d2628c52ecb42029dc6d20b52d787c"
+  license "MIT"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "scdoc" => :build
+  depends_on "tllist" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "harfbuzz"
+  depends_on "pixman"
+
+  def install
+    system "meson", "setup", "build", "-Ddocs=enabled", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+
+    (pkgshare/"example").install Dir["example/*"]
```

Comment:
```suggestion
    pkgshare.install "example"
```
Is this not equivalent?

Reply:
Fixed.